.github/ISSUE_TEMPLATE/bug_report.md

```
@@ -0,0 +1,28 @@
+---
+## **Describe the bug:**
+
+## **Steps to Reproduce:**
+
+### **Expected behavior:**
+
+## **Screenshot(s)/Video(s) for reference:**
+
+## **Debug log:**
+
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)